WordPress/twentytwentyfive
# Update: Set correct footers and header to "News" blog templates

**Description**

Fixes https://github.com/WordPress/twentytwentyfive/issues/591

- Updating the headers and footers as requested in https://github.com/WordPress/twentytwentyfive/issues/591
- Renaming the file `patterns/post-with-left-aligned-content.php` to match conventions used in other single files.

**Screenshots**

N/A 

**Testing Instructions**

- Open the site editor.
- Edit the home template, archive, search results and single post templates to use the ones affected by this PR.
- Confirm that the headers and footers match what was requested in the GH issue.
